src/test/reference_buffers/CueGotoAndStopTest

```diff
@@ -1,512 +1,512 @@
--0.189029,-0.189029
--0.180783,-0.180783
--0.172014,-0.172014
--0.162638,-0.162638
--0.152768,-0.152768
--0.14244,-0.14244
--0.13166,-0.13166
--0.120567,-0.120567
--0.109085,-0.109085
--0.097435,-0.097435
--0.0854568,-0.0854568
--0.0734421,-0.0734421
--0.061223,-0.061223
--0.0490174,-0.0490174
--0.0367553,-0.0367553
--0.0245894,-0.0245894
--0.0124638,-0.0124638
--0.000562549,-0.000562549
-0.0112367,0.0112367
-0.0227116,0.0227116
-0.033957,0.033957
-0.0448621,0.0448621
-0.0553823,0.0553823
-0.0655431,0.0655431
-0.0752199,0.0752199
-0.0844273,0.0844273
-0.0931864,0.0931864
-0.101299,0.101299
-0.109022,0.109022
-0.115976,0.115976
-0.122495,0.122495
-0.128294,0.128294
-0.133508,0.133508
-0.138062,0.138062
-0.142004,0.142004
-0.145218,0.145218
-0.147888,0.147888
-0.149785,0.149785
-0.151116,0.151116
-0.151776,0.151776
-0.151797,0.151797
-0.151202,0.151202
-0.149998,0.149998
-0.148197,0.148197
-0.145836,0.145836
-0.142932,0.142932
-0.139465,0.139465
-0.135557,0.135557
-0.131101,0.131101
-0.126281,0.126281
-0.120981,0.120981
-0.115328,0.115328
-0.109343,0.109343
-0.102985,0.102985
-0.0964258,0.0964258
-0.0895391,0.0895391
-0.0825073,0.0825073
-0.0752537,0.0752537
-0.0679016,0.0679016
-0.0604182,0.0604182
-0.0528749,0.0528749
-0.0453173,0.0453173
-0.03773,0.03773
-0.0302151,0.0302151
-0.0227506,0.0227506
-0.0153856,0.0153856
-0.00817108,0.00817108
-0.00110453,0.00110453
--0.00575781,-0.00575781
--0.0124204,-0.0124204
--0.0188186,-0.0188186
--0.0249674,-0.0249674
--0.0308222,-0.0308222
--0.0363487,-0.0363487
--0.0415993,-0.0415993
--0.0464467,-0.0464467
--0.0509977,-0.0509977
--0.0551309,-0.0551309
--0.0589228,-0.0589228
--0.062319,-0.062319
--0.0653213,-0.0653213
--0.0679459,-0.0679459
--0.0701551,-0.0701551
--0.0719938,-0.0719938
--0.0734239,-0.0734239
--0.0744871,-0.0744871
--0.0751584,-0.0751584
--0.075464,-0.075464
--0.0754148,-0.0754148
--0.0750076,-0.0750076
--0.0742807,-0.0742807
--0.0732235,-0.0732235
--0.0718684,-0.0718684
--0.0702256,-0.0702256
--0.0683241,-0.0683241
--0.0661613,-0.0661613
--0.063794,-0.063794
--0.061195,-0.061195
--0.0584435,-0.0584435
--0.055504,-0.055504
--0.0524478,-0.0524478
--0.0492636,-0.0492636
--0.0459988,-0.0459988
--0.0426599,-0.0426599
--0.0392747,-0.0392747
--0.0358713,-0.0358713
--0.0324669,-0.0324669
--0.0290779,-0.0290779
--0.0257456,-0.0257456
--0.0224537,-0.0224537
--0.0192695,-0.0192695
--0.0161592,-0.0161592
--0.0131881,-0.0131881
--0.0103384,-0.0103384
--0.0076375,-0.0076375
--0.005108,-0.005108
--0.00273407,-0.00273407
--0.000565767,-0.000565767
-0.00142813,0.00142813
-0.00320607,0.00320607
-0.00478506,0.00478506
-0.0061506,0.0061506
-0.00730038,0.00730038
-0.00823528,0.00823528
-0.00895631,0.00895631
-0.0094564,0.0094564
-0.00975609,0.00975609
-0.00984198,0.00984198
-0.00973403,0.00973403
-0.00943226,0.00943226
-0.00894785,0.00894785
-0.00829291,0.00829291
-0.00747693,0.00747693
-0.00651449,0.00651449
-0.00541663,0.00541663
-0.00420177,0.00420177
-0.00288188,0.00288188
-0.00147617,0.00147617
+0,0
```

Comment:
why are these all zeros now?

Reply:
That seems to be wrong. I will have a look.